.github/workflows/cml.yaml

```diff
@@ -0,0 +1,40 @@
+name: test-run
+ 
+on: [push]
+ 
+jobs:
+  run:
+    runs-on: [ubuntu-latest]
```

Comment:
@HarshCasper - I think we can also add:

```diff
runs-on: [ubuntu-latest]
+container: ghcr.io/moja-global/rest_api_gcbm:master
```

Reply:
Agreed.

Reply:
Yes, I will add the container for the workflow.